pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/redigerbar/EndringAvAlderspensjonSivilstand.kt

```diff
@@ -0,0 +1,1127 @@
+                    Nynorsk to " og 22-12.",
+                    English to " and 22-12.",
+                )
+            }
+
+            showIf(
+                kravArsakType.isOneOf(KravArsakType.ALDERSOVERGANG) and regelverkType.isNotAnyOf(
+                    AlderspensjonRegelverkType.AP2025
+                )
+            ) {
+                // hjemmelGarantitillegg§20-20
+                paragraph {
+                    text(
+                        Bokmal to "Vedtaket er gjort etter folketrygdloven § 20-20.",
+                        Nynorsk to "Vedtaket er gjort etter folketrygdlova § 20-20.",
+                        English to "This decision was made pursuant to the provisions of § 20-20 of the National Insurance Act.",
+                    )
+                }
+            }
+        }
+
+        showIf(
+            kravArsakType.isOneOf(KravArsakType.VURDER_SERSKILT_SATS) and saerskiltSatsErBrukt
+        ) {
+            paragraph {
+                text(
+                    Bokmal to "Vedtaket er gjort etter folketrygdloven §§ ",
+                    Nynorsk to "Vedtaket er gjort etter folketrygdlova §§ ",
+                    English to "This decision was made pursuant to the provisions of §§ ",
+                )
+                showIf(regelverkType.isOneOf(AlderspensjonRegelverkType.AP1967)) {
+                    showIf(brukersSivilstand.isOneOf(MetaforceSivilstand.SAMBOER_1_5)) {
+                        text(
+                            Bokmal to "1-5, ",
+                            Nynorsk to "1-5, ",
+                            English to "1-5, "
+                        )
+                    }
+                    text(
+                        Bokmal to "3-2",
+                        Nynorsk to "3-2",
+                        English to "3-2"
+                    )
+                    showIf(saertilleggInnvilget) {
+                        text(
+                            Bokmal to ", 3-3",
+                            Nynorsk to ", 3-3",
+                            English to ", 3-3"
+                        )
+                    }
+                    text(
+                        Bokmal to ", 19-8 og 22-12.",
+                        Nynorsk to ", 19-8 og 22-12.",
+                        English to ", 19-8 and 22-12 of the National Insurance Act."
+                    )
+                }.orShowIf(
+                    regelverkType.isOneOf(
+                        AlderspensjonRegelverkType.AP2011,
+                        AlderspensjonRegelverkType.AP2016
+                    )
+                ) {
+                    showIf(brukersSivilstand.isOneOf(MetaforceSivilstand.SAMBOER_1_5)) {
+                        text(
+                            Bokmal to "1-5, ",
+                            Nynorsk to "1-5, ",
+                            English to "1-5, "
+                        )
+                    }
+                }
+                    text(
```

Comment:
Denne teksten må ett hakk opp slik at den er en del av AP2011/AP2016 logikken. Ellers vil den komme dobbelt for AP1967